graphics/blit-atari.cpp

```diff
@@ -40,24 +41,31 @@ void Surface::create(int16 width, int16 height, const PixelFormat &f) {
 	w = width;
 	h = height;
 	format = f;
-	pitch = w * format.bytesPerPixel;
+	pitch = (w * format.bytesPerPixel + 15) & 0xfff0;
 
 	if (width && height) {
-		if (VgetMonitor() == MON_VGA && Getcookie(C_SupV, NULL) == C_FOUND)
-			pixels = (void*)ct60_vmalloc(width * height * format.bytesPerPixel);
-		else
-			pixels = calloc(width * height, format.bytesPerPixel);
-		assert(pixels);
+		if (VgetMonitor() == MON_VGA && Getcookie(C_SupV, NULL) == C_FOUND) {
+			pixelsUnaligned = (void*)ct60_vmalloc(height * pitch + 15);
```

Comment:
Isn't this already aligned?

Reply:
Unfortunately, only at a 4-byte boundary. :( This call is usually not used for classic buffers but in this case it makes things simpler (as it allows me to detect when both src and dst are able to use the hardware blitter, otherwise to fall back to the `move16` copy).